templates/aws/fips.html

```diff
@@ -184,6 +171,20 @@ <h2>Launch Ubuntu Pro FIPS for AWS</h2>
     </div>
     <div class="row">
       <div class="col-start-large-4 col-9 col-medium-6">
+        <hr class="p-rule--muted" />
+        <div class="row">
+          <div class="col-3 col-medium-3">
+            <h3 class="p-heading--5">Ubuntu Pro FIPS 22.04 LTS</h3>
+          </div>
+          <div class="col-6 col-medium-3">
+            <p>
+              Ubuntu Pro FIPS 22.04 LTS includes an AWS-optimised FIPS-certified 5.15 kernel and other FIPS-certified modules pre-enabled out of the box. Ubuntu Pro FIPS provides extended security maintenance through April 2032.
+            </p>
+            <hr class="is-muted" />
+            <a href="https://aws.amazon.com/marketplace/pp/prodview-l2hkkatnodedk"
+               class="p-button--positive">Launch 22.04 LTS</a>
```

Comment:
```suggestion
               class="p-button--positive">Launch 22.04 LTS (AMD64)</a>
```
Also missing the second button listed in the doc "Launch 22.04 LTS (Graviton) "